frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerLifeCycle.java

```diff
@@ -45,6 +49,30 @@ public WorkerLifeCycle(ConfigManager configManager, Model model) {
         this.model = model;
     }
 
+    private String[] getEnvString(String cwd, String modelPath) {
+        ArrayList<String> envList = new ArrayList<>();
```

Comment:
In java convention, variable declaration should be as closer to first usage as possible.